src/main/java/com/gtnewhorizons/gtnhgradle/modules/ideintegration/IdeaMiscXmlUpdater.java

```diff
@@ -0,0 +1,118 @@
+package com.gtnewhorizons.gtnhgradle.modules.ideintegration;
+
+import org.jdom2.Document;
+import org.jdom2.Element;
+import org.jdom2.input.SAXBuilder;
+import org.jdom2.output.Format;
+import org.jdom2.output.XMLOutputter;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.BufferedWriter;
+import java.io.StringReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class IdeaMiscXmlUpdater {
+
+    /** The default misc.xml for IntelliJ */
+    private static final @NotNull String MISC_XML_DEFAULT = """
+        <?xml version="1.0" encoding="UTF-8"?>
+        <project version="4">
+          <component name="EntryPointsManager">
+            <list size="4">
+              <item index="0" class="java.lang.String" itemvalue="cpw.mods.fml.common.Mod.EventHandler" />
+              <item index="1" class="java.lang.String" itemvalue="cpw.mods.fml.common.eventhandler.SubscribeEvent" />
+              <item index="2" class="java.lang.String" itemvalue="net.minecraftforge.fml.common.eventhandler.SubscribeEvent" />
+              <item index="3" class="java.lang.String" itemvalue="org.spongepowered.asm.mixin.Mixin" />
+            </list>
+          </component>
+          <component name="ExternalStorageConfigurationManager" enabled="true" />
+          <component name="ProjectRootManager" version="2" languageLevel="JDK_17" project-jdk-name="11" project-jdk-type="JavaSDK">
```

Comment:
Not sure if it's a good idea to include this here. I think that will overwrite the default SDK when importing a project. Previously no SDK would be set explicitly.